content/en/project/stories.md

```diff
@@ -3,6 +3,7 @@ title: Our Stories
 weight: 40
 type: docs
 ---
+Many of these weren from posts on one the Interlisp mailing lists, and others written expressedly. What are **you** using Medley for? 
```

Comment:
This sentence doesn't read correctly.

Reply:
leaving stores alone. If Ron started a sentence with And, shame on him.

Reply:
Not an "And..." issue: "weren" should be "were", and "expressedly" (not a word) should be "expressly"

Reply:
Well, "expressedly" is an archaic form, as in:

>Adverb. expressedly (comparative more expressedly, superlative most expressedly) (chiefly nonstandard or archaic) In a way that is expressed or made explicit.

although it is usually a mispelling of expressly.